app/javascript/components/laboratory/Laboratory.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
-import { Button, Col, Form, Modal, Row } from 'react-bootstrap';
-
+import { Form, Row, Col, Button, Modal } from 'react-bootstrap';
```

Comment:
was this an unintentional change?

Reply:
Nope. i'll take a look. i had done some work in that file, and then reverted things, so i'll make sure it's what it should be